ui/webui/test/check-review

```diff
@@ -50,5 +50,18 @@ class TestReview(MachineCase):
         # Pixel test the review step
         b.assert_pixels("#app", "review-step-basic", ignore=["#betanag-icon"])
 
+    def testConfirmation(self):
+        b = self.browser
+        i = Installer(b)
+
+        i.open()
+
+        for page in i.steps[:-1]:
+            if i.steps[i.get_current_page_id()] != i.review_id:
+                i.next()
+            else:
+                i.begin_installation(should_fail=True, confirm_erase=False)
+                i.begin_installation()
```

Comment:
This test will actually start the installation, therefore it's becoming a destructive tests.
TestReview test class is marked as non-destructive, so this is also marked as non-destructive, which means it will share the VM with some other test, and make it fail. This is the case that we see in the logs.

The 'confirm' case is already tested in the TestInstallationProgress, as we need to confirm in order to start the installation.

Reply:
I've already noticed the problem with @nondestructive. Note that this test case should verify that the installation doesn't start if you don't confirm the erase.

Reply:
Please read the guidelines https://cockpit-project.org/external/source/test/README
especially the section for `Guidelines for writing tests`

Reply:
It's fine to non-destructive tests to be very small, as their 'setup' costs nothing but the destructive tests need to be longer.
This test indeed checks that the installation won't start without the 'confirmation dialog', and till this part the test is non-destructive and therefore fine to be so small.

However, the final begin-installation() call makes the test destructive, and also offers nothing more in matters of code coverage, compared to TestInstallationProgress